### ISSUE TITLE
`aria-*` attributes are not passed to the underlying component

### ISSUE THREAD

**Issue by tuananhlai** [tuananhlai/hinagata-vue#17]
```
<Button aria-label="Label" /> // doesn't work
```

**Comment by tuananhlai** [comment]
Since Vue normalizes passed attributes into camel case, field name of component props should only be in camel case.
